### PR TITLE
converted the download .zip file for windows users into simple zip

### PR DESCRIPTION
**Pull request by knuxus**
Since the current download button does give you the tar.gz file which is not optimal for windows users.

### CODE REVIEW

**Comment by despo** [comment]
@knuxus could you add it as an alternative option instead of replacing the current link.

**Comment by knuxus** [comment]
Edited now, if you wish to correct my spelling/meaning go ahead, hope it helps

File: html/lesson3/tutorial.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ We will also be explaining in more detail elements that we mentioned in our prev
 
 ### Required files
 
-Download the files required to begin working through the tutorial from [here](https://gist.github.com/despo/7328342/download)
+Download the files required to begin working through the tutorial from from [here](https://gist.github.com/despo/7328342/download) or for Windows users if you are having trouble extracing, you can use this [link](assets/lesson3.zip)
```
Comment (jkbits1) [inline html/lesson3/tutorial.md]: Looks good :smile:  2 minor points - there is an extra "from" and "extrac(t)ing" is missing the "t"

**Comment by knuxus** [comment]
@jkbits1 and @despo , corrected, merge if you wish.

**Comment by despo** [comment]
thanks @knuxus :+1:

**Pull request merged**